collector/src/main/java/com/bitmovin/analytics/data/SimpleEventDataDispatcher.java

```diff
@@ -61,7 +69,7 @@ public void disable() {
     @Override
     public void add(EventData eventData) {
         eventData.setSequenceNumber(this.sampleSequenceNumber++);
-
+        eventData.setKey(licenser.getAnalyticsLicense());
```

Comment:
I don't see where the Dispatcher is disabled if no license was returned by the Licensing? If I'm reading this correctly it would send out a bunch of samples with `null` as `key` and our backend would obviously reject these requests.

Reply:
This is already handled by the `detach` method.